test/functional/blockchain.py

```diff
@@ -49,6 +49,9 @@ def _test_gettxoutsetinfo(self):
         assert_equal(res['transactions'], 200)
         assert_equal(res['height'], 200)
         assert_equal(res['txouts'], 200)
+        size = res['disk_size']
+        assert size > 6400
```

Comment:
In commit "Report on-disk size in gettxoutsetinfo"

Could assert `6400 < size < 64000`